File: 1.2 Patterns/10-90deg-rot-triangle.py
row = 5
for i in range (0, row):
    for j in range (0, i + 1):
        print("*", end ="")
    print("")                           
for i in range (row - 1, -1, -1):
    for j in range (i):
        print("*", end ="")
    print("") 

File: 1.2 Patterns/10-90deg-rot-triangle.cpp
#include<iostream>
using namespace std;
int main(){
    int row = 5;
    for(int i = 0 ; i < row ; i++){
        for(int j = 0 ; j <= i ; j++){
            cout<<"*";
        }
        cout<<"\n";
    }
    
    for(int i = row - 1 ; i > 0 ; i--){
        for(int j = 0 ; j < i ; j++){
            cout<<"*";
        }
        cout<<"\n";
    }
    return 0;
}